code/web/src/modules/user/Profile.js

```diff
@@ -34,8 +35,14 @@ const Profile = (props) => (
 
     <Grid>
       <GridCell style={{ padding: '2em', textAlign: 'center' }}>
+        <ProfilePicture />
         <H4 style={{ marginBottom: '0.5em' }}>{props.user.details.name}</H4>
         <p style={{ color: grey2, marginBottom: '2em' }}>{props.user.details.email}</p>
+
+        <Link to={userRoutes.editProfile.path}>
+          <Button style={{ margin: '0.5em'}} theme="primary">Edit Profile</Button>
+        </Link>
+
```

Comment:
Thank you for adding the profile picture to this component so it will display when they view their profile.